### PR TITLE
RHMAP-10216 Remove permission change for container recycle

### PR DESCRIPTION
**Pull request by aidenkeating**
This removes a permission change which changes the permissions on
the underlying pv directory.

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by aidenkeating** [comment]
@wtrocki Could you take a look please?

File: bin/start.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 LOG_LEVEL=${LOG_LEVEL:-INFO}
 
 echo "Setting sshd LogLevel to $LOG_LEVEL"
-sed -i "s/#LogLevel INFO/LogLevel ${LOG_LEVEL:-INFO}/" /etc/ssh/sshd_config
+sed -i "s/#LogLevel INFO/LogLevel ${LOG_LEVEL:-DEBUG2}/" /etc/ssh/sshd_config
```
Comment (aidenkeating) [inline bin/start.sh]: This needs to be `INFO`

Comment (wtrocki) [inline bin/start.sh]: Revert that change.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-2.1.1.pre
+2.1.2.pre
```
Comment (wtrocki) [inline VERSION]: This represents official gitlab shell version. I think we should not change that value. We do not to reference this value anywhere - deployment is done using git branch.

Comment (aidenkeating) [inline VERSION]: Ah ok cool, I didn't know that, will revert now

**Comment by wtrocki** [comment]
Reviewed

**Comment by aidenkeating** [comment]
retest this please

**Pull request merged**